From 8abae7c7f37f364f1faec6bb4d1ecbc404730ff1 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Mon, 18 Aug 2014 19:56:54 -0400
Subject: [scsi] qla2xxx: Remove mapped vp index iterator macro dead code

Message-id: <1408391817-6389-67-git-send-email-cdupuis@redhat.com>
Patchwork-id: 87944
O-Subject: [PATCH RHEL 7.1 e-stor 66/69] qla2xxx: Remove mapped vp index iterator macro dead code.
Bugzilla: 1089346
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Himanshu Madhani <himanshu.madhani@qlogic.com>

scsi commit id 5921cda6c1a402bacbfa5c97bbb7039eb0fd9dd8

Signed-off-by: Himanshu Madhani <himanshu.madhani@qlogic.com>
Signed-off-by: Saurav Kashyap <saurav.kashyap@qlogic.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla2xxx/qla_fw.h b/drivers/scsi/qla2xxx/qla_fw.h
index 073d1f4..b34cd3c 100644
--- a/drivers/scsi/qla2xxx/qla_fw.h
+++ b/drivers/scsi/qla2xxx/qla_fw.h
@@ -1134,13 +1134,6 @@ struct device_reg_24xx {
 #define MIN_MULTI_ID_FABRIC 64 /* Must be power-of-2. */
 #define MAX_MULTI_ID_FABRIC 256 /* ... */
 
-#define for_each_mapped_vp_idx(_ha, _idx)  \
- for (_idx = find_next_bit((_ha)->vp_idx_map, \
-  (_ha)->max_npiv_vports + 1, 1);  \
-     _idx <= (_ha)->max_npiv_vports;  \
-     _idx = find_next_bit((_ha)->vp_idx_map, \
-  (_ha)->max_npiv_vports + 1, _idx + 1)) \
-
 struct mid_conf_entry_24xx {
  uint16_t reserved_1;
 
-- 
1.7.1